src/replica/replica_disk_migrator.cpp

```diff
@@ -148,12 +152,141 @@ bool replica_disk_migrator::check_migration_args(const replica_disk_migrate_requ
     return true;
 }
 
-// TODO(jiashuo1)
 // THREAD_POOL_REPLICATION_LONG
-void replica_disk_migrator::migrate_replica(const replica_disk_migrate_request &req) {}
+void replica_disk_migrator::do_disk_migrate_replica(const replica_disk_migrate_request &req)
```

Comment:
I think rename the func name to migrate_replica_disk is a little better

Reply:
maybe I should revert to `migrate_replica`,  ` replica_disk_migrator::` has indicated it migrate replica on disk

Reply:
> maybe I should revert to `migrate_replica`, ` replica_disk_migrator::` has indicated it migrate replica on disk

Yes. You make this name too verbose.